File: 1470 shuffle-the-array/solution.cs
public class Solution {
    public int[] Shuffle(int[] nums, int n) {
         int[] res=new int[nums.Length];
        for(int i=0;i<n;i++) {
           res[2*i]=nums[i];
           res[2*i+1]=nums[i+n];
        }
        return res;
    }
}

File: 1470 shuffle-the-array/solution.rs
impl Solution {
    pub fn shuffle(nums: Vec<i32>, n: i32) -> Vec<i32> {
        let n = n as usize;
        let mut res = vec![0; nums.len()];

        for i in 0..n {
            res[2 * i] = nums[i];
            res[2 * i + 1] = nums[i + n];
        }

        res
    }
}
